tensorflow/lite/examples/python/label_image.py

```diff
@@ -19,13 +19,11 @@
 from __future__ import print_function
 
 import argparse
+import time
 
 import numpy as np
-
-from PIL import Image
-
 import tensorflow as tf # TF2
-
+from PIL import Image
```

Comment:
Is this needed?

Reply:
yes, to read an image and resize it.

Reply:
Oh yeah, I was wondering why this one marked as added - and I noticed that this is a more-than-a-year-old PR :( (sorry about that)